src/openzaak/components/catalogi/api/serializers/besluittype.py

```diff
@@ -107,3 +107,23 @@ class Meta:
             M2MConceptCreateValidator(["informatieobjecttypen"]),
             M2MConceptUpdateValidator(["informatieobjecttypen"]),
         ]
+
+
+class BesluitTypePublishSerializer(serializers.HyperlinkedModelSerializer):
+    class Meta:
+        model = BesluitType
+        extra_kwargs = {
+            "catalogus": {"lookup_field": "uuid"},
+            "begin_geldigheid": {"source": "datum_begin_geldigheid"},
+            "einde_geldigheid": {"source": "datum_einde_geldigheid"},
+        }
+        fields = (
```

Comment:
it looks like publish serializers you only `concept` field, so other defined fields are not needed

Reply:
Ended up create a separate `GeldigheidPublishValidator` that gets most fields from the instance